langserver/langserver.go

```diff
@@ -12,12 +12,15 @@ import (
 	"github.com/creachadair/jrpc2"
 	"github.com/creachadair/jrpc2/channel"
 	"github.com/creachadair/jrpc2/server"
+
+	"github.com/hashicorp/terraform-ls/internal/terraform/discovery"
 	"github.com/hashicorp/terraform-ls/langserver/session"
 )
 
 type langServer struct {
 	srvCtx     context.Context
 	logger     *log.Logger
+	tfPath     discovery.DiscoveryFunc
```

Comment:
Nitpick, but do you mind calling this `tfDiscoFunc` or something similar that makes it more obvious that this variable is a function and not a string?

I suppose the other `tfPath` in `service.go` is a forgotten relic that should also be changed - especially once https://github.com/hashicorp/terraform-ls/issues/69 is addressed. I'm happy to address it as part of #69 though - no biggie.